Watchman.Engine/Generation/AlarmBuilder.cs

```diff
@@ -98,7 +99,18 @@ private AlarmValues MergeValueOverrides(string key, Dictionary<string, AlarmValu
 
             return result;
         }
-        
+
+        public string GetAlarmDescription(AlertingGroupParameters groupParameters)
+        {
+            var suffix = string.IsNullOrWhiteSpace(groupParameters.Description)
+                ? null
+                : $" ({groupParameters.Description})";
+
+            var description = $"{AwsConstants.DefaultDescription}. Alerting group: {groupParameters.Name}{suffix}";
+
+            return description;
```

Comment:
👍 We need the default text to identify that this alarm was created by Watchman.
Adding th alarting group looks useful.
Adding custom text if it's present is also good.
Fine by me